doc/source/reference/general_utility_functions.rst

```diff
@@ -30,6 +30,13 @@ Testing functions
    testing.assert_index_equal
    testing.assert_extension_array_equal
 
+Bug report function
```

Comment:
I'd put this at the bottom of this file.

Reply:
@TomAugspurger, I did it.